src/http.js

```diff
@@ -110,7 +110,6 @@ class Http {
 		}
 
 		return promise
-		.then(res => (res.ok || res.status === 302) ? res : res.json())
```

Comment:
Doesn't this break the ratelimit [handling code](https://github.com/simplyGits/MagisterJS/pull/98/files#diff-31b367d1856df8608494b65123d57acdR118)? If I remember correctly this expects an object (on an error)

Reply:
Yep, it was really broken actually. Don't know if I like the current approach either but it works. Might be something to consider being rewritten before v2 final.